### PR TITLE
Add four screen shots for illustration purposes

### PR DESCRIPTION
**Pull request by uwesterh**

### CODE REVIEW

File: doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst [hunk 1]
```diff
@@ -30,9 +30,16 @@ Roslaunch the launch file to run the code directly from moveit_tutorials::
 Expected Output
 ^^^^^^^^^^^^^^^
 
+sdf
```
Comment (davetcoleman) [inline doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst]: is this a typo?

**Comment by uwesterh** [comment]
Yes, this is a typo. Sorry for that.

File: doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst (hunk 1 above)
Comment (davetcoleman) [inline doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst]: Will merge once fixed

Comment (davetcoleman) [inline doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst]: nvm, will fix now

**Pull request merged**